ui/v2.5/src/components/Performers/PerformerDetails/PerformerEditPanel.tsx

```diff
@@ -725,7 +725,7 @@ export const PerformerEditPanel: React.FC<IPerformerDetails> = ({
     );
   }
 
-  function renderTextField(field: string, title: string) {
+function renderTextField(field: string, title: string, placeholder?: string) {
```

Comment:
```suggestion
  function renderTextField(field: string, title: string, placeholder?: string) {
```

to appease prettier (and fix the build)

Reply:
👍